### DISCUSSION TITLE
源文件名称/Source File Name

### DISCUSSION THREAD

**Discussion by ZSY4** [immich-app/immich#6212] [Q&A]
请问这个手机图片上传后在服务器里看图片名字被改成了乱码，能不能设置不改名啊？
Excuse me, after uploading this mobile phone picture, the picture name was changed into garbled code in the server. Can you set not to change your name
另外一个问题，服务器中的目录能不能根据照片时间创建这种存储结构？
「2024
-1
-01
-pictures1」
Another question is, can the directories in the server create this storage structure based on the time of the photos?
「2024
-1
-01
-pictures1」

**Comment by bo0tzz** [comment]
https://immich.app/docs/administration/storage-template
